www/source/tutorials/get-started/demo/responses/hab-monitor-myrubyapp.txt

```diff
@@ -1 +1 @@
-hab-sup(MN): dwrede/myrubyapp/0.1.0/20170520062324, state:up, time:PT234.585418865S, pid:584, group:myrubyapp.default, style:transient
```

Comment:
Tutorials are different than blog posts, though... but should probably be intentionally refreshed on a regular cadence to ensure they're consistent with the latest software. Given the aforementioned formatting changes to the status command, it's probably time to do that.

I guess if we just remove `transient`here now, that takes away an unnecessary concept that new users would have to contend with. Much easier to explain that the output format of a command changed, than to explain a concept that no longer exists.